expenses/expensesRouter.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
```

Comment:
For the way you're organizing the application you're having to require express in quite a few different files.  Not necessarily a terrible thing just something to be aware of.